pages/app/SplashDetail.js

```diff
@@ -146,162 +188,229 @@ class SplashDetail extends Page {
     });
 
     return (
-      <div
-        title="Resource Watch"
-        className={pageClass}
-      >
-        <Head
-          title="SplashDetail page"
-          description="SplashDetail page description"
-        />
-        <Icons />
-        <Header
-          hideSkip
-          showEarthViewLink
-        />
-        {hasIntro &&
-          <div className={classnames('intro-container', `-${introOpened ? 'opened' : 'closed'}`)}>
-            <div className={classnames('text-container', `-${introOpened ? 'opened' : 'closed'}`)}>
-              {selectedPanorama.intro}
-            </div>
-            <button
-              type="button"
-              className={classnames('l-sidebar-toggle', 'btn-toggle', `-${introOpened ? 'opened' : ''}`)}
-              onClick={this.handleToggleIntro}
-            >
-              <Icon
-                className={classnames('-little', `-${introOpened ? 'left' : 'right'}`)}
-                name={`icon-arrow-${introOpened ? 'left' : 'right'}`}
-              />
-            </button>
-          </div>
-        }
-        {selectedHotspot &&
-          <div className="hotspot-section">
-            <div
-              className="selected-hotspot-container"
-              role="button"
-              tabIndex={-1}
-              onClick={this.handleCloseRightMenu}
-            >
-              <img src={selectedHotspot.imageSelected} alt={selectedHotspot.title} />
+      <div>
+        {/* Mobile Splash details page */}
+        <MediaQuery
+          maxDeviceWidth={breakpoints.medium}
```

Comment:
If you use maxWidth, remember to do this `breakpoints.medium - 1`. If you don't that you will trigger this at 1024px. But you want to do it at 1023px

Reply:
Done.